packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -41,9 +45,8 @@ class Layout extends React.Component {
       isMobileNavOpen: !this.state.isMobileNavOpen,
     });
   }
-
   render() {
-    const { nav, helpItems, intl, ...customProps } = this.props;
+    const { nav, helpItems, intl, siteAlert, ...customProps } = this.props;
```

Comment:
I just confirmed with our SD that we want the AlertBanner to be positioned across the entire top of the window, extending over the Nav.

Reply:
Would like to have a visible separation between individual sitAlert elements.

Reply:
Can we attach a screen capture for desktop and mobile views.

Reply:
to better represent how we expect to use the component from within DEx, can you add  
    type={Alert.Opts.Types.INFO}   to the alert props.

![image](https://user-images.githubusercontent.com/13628009/32116040-117cf16a-bb17-11e7-8358-34d9208a8ac7.png)